app/Console/Commands/ImportPatreonPledgers.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Patreon;
+use App\Services\Patreon\PatreonApi;
+use App\Services\Patreon\Resources\Pledge;
+use App\Services\Patreon\Resources\Reward;
+use Illuminate\Console\Command;
+
+class ImportPatreonPledgers extends Command
+{
+    protected $signature = 'import:patreon-pledgers';
+
+    protected $description = 'Import pledgers from Patreon';
+
+    /** @var \App\Services\Patreon\PatreonApi */
+    protected $api;
+
+
+    public function __construct(PatreonApi $api)
+    {
+        $this->api = $api;
+
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        $this->info('Importing pledgers from Patreon...');
+
+        $campagin = $this->api->campaigns()->first();
+
+        $validRewards = $campagin->rewards->filter(function (Reward $reward) {
```

Comment:
Reward under the specified amount are also "valid", so let's renamed the variable to something more correct. Maybe let's just use `$rewards`.